config.yml

```diff
@@ -30,6 +34,10 @@ scenario_params_list:
    epoch_count: 
      - 38
    methods:
-     - ["Shapley values", "Independant scores", "TMCS"]
+     - "Shapley values"
```

Comment:
Need to put them inside a list (like for the quick debug config) otherwise it crashes.

Actually I think it will make more sense, if we could write the methods like this (and not have to put them into a list) but this can be the matter of another PR...

Reply:
OK